docs/superpowers/plans/2026-04-11-phase2-service-layer.md

```
@@ -0,0 +1,563 @@
+- [ ] **Step 1: Full TypeScript check**
+
+Run: `cd apps/native && npx tsc --noEmit 2>&1 | head -40`
+
+Expected: No errors in any new files (`api-client.ts`, `session.service.ts`, all hooks, both stores).
+
+- [ ] **Step 2: Verify file structure**
+
+Run: `cd apps/native && find lib services hooks stores -type f -name "*.ts" | sort`
+
+Expected output:
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)